pom.xml

```diff
@@ -435,7 +435,7 @@
 			<plugin>
 				<groupId>org.pitest</groupId>
 				<artifactId>pitest-maven</artifactId>
-				<version>1.5.2</version>
+				<version>1.6.4</version>
```

Comment:
any reason not to use the latest version 1.7.5?

Reply:
Hello, yeah you're right, was only trying 1.6.4 initially since that was the earliest version that worked. Changed it accordingly again, it works fine now